Client.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -77,7 +77,7 @@
       "location" : "https://github.com/nbhasin2/MappaMundi.git",
       "state" : {
         "branch" : "master",
-        "revision" : "a125309d71dffbeae2efe1df24b0ee9fa0a771a4"
+        "revision" : "7fcd2b7910550227088480490ac19ea66044b3d1"
```

Comment:
This reverts https://github.com/mozilla-mobile/firefox-ios/pull/11870 🤔

Reply:
XCode did this automatically and I have no idea how to undo it 🤔

Reply:
Fixed.